src/osd/OSD.cc

```diff
@@ -5115,7 +5115,7 @@ void OSD::_collect_metadata(map<string,string> *pm)
   (*pm)["hb_back_addr"] = stringify(hb_back_server_messenger->get_myaddr());
 
   // backend
-  (*pm)["osd_objectstore"] = cct->_conf->osd_objectstore;
+  (*pm)["osd_objectstore"] = store->store_type();
```

Comment:
also, please do Fixes: tracker url instead of the #number syntax or else github thinks you're talking about a github PR/issue.